tests/test_domain_objects/back_office_position_test.py

```diff
@@ -0,0 +1,25 @@
+import sys
+from test_domain_objects import shared_tests as shared
+from test_domain_objects import helper_methods as helper
+
+
+sys.path.insert(0, 'tests/')
+
+
+def test_back_office_positions():
+    """ Ensure all generated back office position attributes adhere to their
+    specification. Dependencies on instruments. """
+
+    records, domain_obj = helper.set_up_back_office_position_tests()
+    for record in records:
+        shared.cusip_exists(record)
+        shared.position_type_valid(record)
+        shared.knowledge_date_valid(record)
+        if (record['position_type'] == 'SD'):
```

Comment:
All these tests should have a check on the number of rows just incase we add or remove fields. When we eventually make the number of dummy fields confiurable, we should have the config used for the test be available to make sure the tests still pass even when the number of dummy fields is changed

Reply:
Issue raised: #59